modules/navier_stokes/doc/intro_slides/slides/index.md

```diff
@@ -0,0 +1,227 @@
+
+!equation id=enthalpy-eq
+\frac{\partial \gamma \rho c_p T}{\partial t} + \nabla \cdot \left(  \rho c_p T \vec{u}\right)
+= \nabla \cdot \left(\kappa_f \nabla T \right) - \alpha (T - T_s) + S_{ext}
+
+!row!
+!col! width=30%
+
+!style! fontsize=80%
+- $c_p$ - specific heat
+- $T$ - fluid temperature
+- $\kappa_f$ - effective thermal conductivity
+!style-end!
+
+!col-end!
+
+!col! width=60%
+
+!style! fontsize=80%
+- $\alpha$ - volumetric heat-transfer coefficient (using correlations)
+- $T_s$ - solid temperature
+- $S_{ext}$ - volumetric source term
+!style-end!
+
+!col-end!
+
+!row-end!
+
+!---
+
+# The Navier-Stokes equations (contd.)
+  style=font-size:26pt
+
+The equations are extended with initial and boundary conditions:
+
+- Few examples:
+
+  - Inlet: $\vec{u}=\vec{u}_{inlet}, \quad T=T_{inlet}, \quad \vec{r} \in \partial\Omega_{inlet}$
+  - Outlet: $p = p_{outlet}, \quad \vec{r} \in \partial\Omega_{outlet}$
+  - No-slip, insulated walls: $\vec{u}=\vec{0}, \quad \kappa_f \nabla T \cdot \vec{n} = 0,  \quad \vec{r} \in \partial\Omega_{wall}$
+  - Heat flux on the walls: $\quad -\kappa_f \nabla T \big|_{\partial \Omega} \cdot \vec{n} = 0,  \quad \vec{r} \in \partial\Omega_{heated}$
+  - Free slip on walls: $\vec{u} \cdot \vec{n} = 0, \quad \vec{r} \in \partial\Omega_{slip}$
+  - ...
+
+!---
+
+# Useful Acronyms
+
+Good to know before navigating source code and documentation:
+
+!row!
+!col! width=50%
+
+- +NS+ - +N+avier-+S+tokes
+- +FV+ - +F+inite +V+olume
+- +I+ - +I+ncompressible
+- +WC+ - +W+eakly-+C+ompressible
+- +C+ - +C+ompressible
+- +P+ - +P+orous-Medium
+
+!col-end!
+
+!col! width=50%
+
+Few Examples:
+
+- +PINSFV:+ +P+orous-Medium +I+ncompressible +N+avier-+S+tokes +F+inite +V+olume
+- +WCNSFV:+ +W+eakly-+C+ompressible +N+avier-+S+tokes +F+inite +V+olume
+
+!col-end!
+
+!row-end!
+
+!---
+
+# Building Input Files
+style=font-size:28pt
+
+The building blocks in MOOSE for terms in the PDEs are +Kernels+ for FE or +FVKernels+ for FV:
+
+!style! fontsize=80%
+!equation
+\underbrace{\frac{\partial \rho  \vec{u}}{\partial t}}_{\text{PINSFMomentumTimeDerivative}}
++ \underbrace{\nabla \cdot \left(\gamma^{-1}  \rho \vec{u} \otimes \vec{u}\right)}_{\text{PINSFVMomentumAdvection}} =
+
+!equation
+\underbrace{\nabla \cdot \left(\gamma \mu_\text{eff} \left(\nabla\vec{u}_I + \left(\nabla \vec{u}_I\right)^T-\frac{2}{3}\nabla\cdot\vec{u}_I\mathbb{I}\right)\right)}_{\text{PINSFVMomentumDiffusion}}
+\underbrace{-\gamma\nabla p}_{\text{PINSFVMomentumPressure}}
++ \underbrace{\gamma\rho \vec{g}}_{\text{PINSFVMomentumGravity~}}
+\underbrace{-  W  \vec{u}_I}_{\text{PINSFVMomentumFriction}}
+
+The building blocks in MOOSE for boundary conditions are +BCs+ for FE or +FVBCs+ for FV:
+
+- Inlet Velocity: $\text{INSFVInletVelocityBC}$
+- Inlet Temperature: $\text{FVFunctionDirichletBC}$
+- Outlet pressure: $\text{INSFVOutletPRessureBC}$
+- Heatflux: $\text{FVFunctionNeumannBC}$
```

Comment:
this one isnt right

Reply:
Which one? I got this from the actin, do we need to change that too?

```
else if (_energy_wall_types[bc_ind] == "heatflux")
    {
      const std::string bc_type = "FVFunctionNeumannBC";
      InputParameters params = _factory.getValidParams(bc_type);
      params.set<NonlinearVariableName>("variable") = _fluid_temperature_name;
      params.set<FunctionName>("function") = _energy_wall_function[bc_ind];
      params.set<std::vector<BoundaryName>>("boundary") = {_wall_boundaries[bc_ind]};

      _problem->addFVBC(bc_type, _fluid_temperature_name + "_" + _wall_boundaries[bc_ind], params);
    }
```